src/tactic/lint/frontend.lean

```diff
@@ -104,14 +104,31 @@ list.reverse $ rb_lmap.values $ rb_lmap.of_list $
   results.fold [] $ λ decl linter_warning results,
     (((e.decl_pos decl).get_or_else ⟨0,0⟩).line, (decl, linter_warning)) :: results
 
-/-- Formats a linter warning as `#print` command with comment. -/
+/-- Formats a linter warning as `#check` command with comment. -/
 meta def print_warning (decl_name : name) (warning : string) : format :=
 "#check @" ++ to_fmt decl_name ++ " /- " ++ warning ++ " -/"
 
+/--
+Prints a workflow command to emit an error understood by github in an actions workflow.
+This enables CI to tag the parts of the file where linting failed with annotations, and makes it
+easier for mathlib contributors to see what needs fixing.
+See https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-error-message
+-/
+meta def print_workflow_command (env : environment) (decl_name : name) (warning : string) :
+  option string :=
+do
+  po ← env.decl_pos decl_name,
+  ol ← env.decl_olean decl_name,
+  return sformat!"\n::error file={ol},line={po.line},col={po.column}::{decl_name} - {warning}"
```

Comment:
What happens with multi-line warnings like from simp_nf?

Reply:
We need to escape the newlines for best results I believe, I'll add this!
I had convinced myself that no linter produced such warnings, but of course I missed that one :D!

Reply:
Just to make sure, could you add a test that fails `simp_nf` to test the multiline handling?

Reply:
A test for us to look at in the files tab there already, I am awaiting CI with fingers crossed to see if it works.
Or do you want an actual test case? In which case what format do you think it should take? Its kinda difficult to test something whose visible effect only appears on github, but I could add some synthetic ones to make sure nobody randomly changes these functions somehow?

Reply:
Looks like it works except I escaped too much now and github didn't unescape the colon afterwards

Reply:
Oh, sorry -- I was being dumb and misread the test. Look good. I don't think there's any reason to put this in `tests/`.